ivy/functional/backends/jax/experimental/layers.py

```diff
@@ -235,7 +238,7 @@ def max_pool2d(
     )
 
     if data_format == "NCHW":
-        res = jnp.transpose(res, (0, 3, 1, 2))
+        return jnp.transpose(res, (0, 3, 1, 2))
```

Comment:
You didn't completely revert the changes 😅

Reply:
:neutral_face: